src/resources/formats/revealjs/styles.html

```diff
@@ -71,4 +71,28 @@
   .reveal .slide figure > figcaption {
     font-size: 0.5em
   }
+
+  .reveal .slide img.stretch + p.caption {
```

Comment:
Remove the `text-align` as we are no longer centering figure captions

Reply:
Removed and the other rule is moved to quarto.scss